pkg/mpc-examples/contracts/04-pause-unpause-controller/README.md

```diff
@@ -0,0 +1,42 @@
+# NullController
+
+## Summary
+PauseUnpauseController is a Managed Pool Controller that has the ability to enable & disable swaps of a managed pool. The PauseUnpauseController exists to work as a mental model for how enabling & disabling swaps of a managed pool can work. PauseUnpauseControllerFactory demonstrates a factory that can deploy both a Managed Pool and a controller that are both aware of each other without using a separate `initialize()` function.
+
+## Details
+The PauseUnpauseController has two functionalities. It can disable the swaps going through a managed pool. 
+The second functionality is the enabling of pool swaps. This behaviour is expected to be used once swapping of a managed pool has been paused and should be enabled again. The two possible ways to unpause the pool are:
+
+- unsafe unpause (no gradualSwapFeePercentage change)
+- safe unpause (gradualSwapFeePercentage change)
+
+The safe unpause mechanism can be used to mitigate arbitrage losses occuring if pool swaps are enabled and the market has moved since the pool was paused. Increasing the swapFeePercentage during the safe unpause ensures that arbitrage opportunities are captured by the LPs of the managed pool via swap fees.
+
```

Comment:
Great job explaining the reasoning behind why to use safe unpause vs unsafe unpause.